### PR TITLE
fix bad deprecated_api link

### PR DESCRIPTION
**Pull request by randyh62**
corrected broken link

### CODE REVIEW

File: docs/how-to/hip_porting_driver_api.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The CUDA Runtime API unifies the Context API with the Device API. This simplifie
 HIP provides a context API to facilitate easy porting from existing Driver codes.
 In HIP, the Ctx functions largely provide an alternate syntax for changing the active device.
 
-Most new applications will prefer to use `hipSetDevice` or the stream APIs , therefore HIP has marked hipCtx APIs as **deprecated**. Support for these APIs may not be available in future releases. For more details on deprecated APIs please refer [HIP deprecated APIs](https://github.com/ROCm/HIP/blob/develop/docs/reference/deprecated_api_list.md).
+Most new applications will prefer to use `hipSetDevice` or the stream APIs, therefore HIP has marked `hipCtx` APIs as **deprecated**. Support for these APIs may not be available in future releases. For more details on deprecated APIs please refer [HIP deprecated APIs](../reference/deprecated_api_list).
```
Comment (lpaoletti) [inline docs/how-to/hip_porting_driver_api.md]: Seems odd - how can an application "prefer"?
Perhaps this can be reworded.

Comment (lpaoletti) [inline docs/how-to/hip_porting_driver_api.md]:
```suggestion
Most new applications will prefer to use `hipSetDevice` or the stream APIs, therefore HIP has marked `hipCtx` APIs as **deprecated**. Support for these APIs may not be available in future releases. For more details about deprecated APIs, see [HIP deprecated APIs](../reference/deprecated_api_list).
```

**Review by lpaoletti — COMMENTED** [review]
Perhaps these highlighted titles can align with the actual topic titles.
image

File: docs/how-to/hip_porting_driver_api.md (hunk 1 above)
Comment (randyh62) [reply docs/how-to/hip_porting_driver_api.md]: Reworded

**Comment by randyh62** [comment]
I need two approvals to squash and merge and cherry-pick to published docs.

**Comment by yhuiYH** [comment]
> I need two approvals to squash and merge and cherry-pick to published docs.

@jujiang-del to help

**Pull request closed without merging**